### PR TITLE
Jetty docs cleanup

### PR DESCRIPTION
**Pull request by eriwen**
This puts the jetty page back in the user guide with references to use the Gretty plugin

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/webTutorial.adoc [hunk 1]
```diff
@@ -18,12 +18,12 @@
...
 ====
- 
+
 This chapter is a work in progress.
- 
+
 ====
 
-This chapter introduces the Gradle support for web applications. Gradle provides two plugins for web application development: the War plugin and the Jetty plugin. The War plugin extends the Java plugin to build a WAR file for your project. The Jetty plugin extends the War plugin to allow you to deploy your web application to an embedded Jetty web container.
+This chapter introduces the Gradle support for web applications. Gradle recommends two plugins for web application development: the War plugin and the https://plugins.gradle.org/plugin/org.akhikhl.gretty[Gretty plugin]. The War plugin extends the Java plugin to build a WAR file for your project. The Gretty plugin extends the War plugin to allow you to deploy your web application to an embedded Jetty web container.
```
Comment (wolfs) [inline subprojects/docs/src/docs/userguide/webTutorial.adoc]: ❌ I am not so sure about "The Gretty plugin extends the War plugin". How about "The Gretty plugin allows you to..."?

Comment (wolfs) [inline subprojects/docs/src/docs/userguide/webTutorial.adoc]: ❌ Could you add a link to the userguide page of the `War` plugin, too?

Comment (eriwen) [inline subprojects/docs/src/docs/userguide/webTutorial.adoc]: Done, thanks.

Comment (eriwen) [inline subprojects/docs/src/docs/userguide/webTutorial.adoc]: There's a link at the bottom. I'm not sure how to link to that page in a way where the link text isn't dictated.

Comment (wolfs) [inline subprojects/docs/src/docs/userguide/webTutorial.adoc]: @eriwen Like this: `<<war_plugin,War plugin>>` ? If you don't want to add it that's fine, too. It just felt better aligned with having a link to the Gretty plugin, too.

Comment (eriwen) [inline subprojects/docs/src/docs/userguide/webTutorial.adoc]: Ah, I didn't know the syntax there. I've done this in another commit, thanks.
